### PR TITLE
Changelog 1.10.1

### PR DESCRIPTION
**Pull request by 6543**
as title

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,27 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix panic when diff (#9187) (#9193)
+  * Fix #9151 - smtp logger configuration sendTos should be an array (#9154) (#9157)
+  * Always Show Password Field on Link Account Sign-in Page (#9150)
+  * Create PR on Current Repository by Default (#8670) (#9141)
+  * Fix race on indexer (#9136) (#9139)
+  * Fix reCAPTCHA URL (#9119)
+  * Hide migrated credentials (#9098)
+  * Update golang.org/x/crypto vendor to use acme v2 (#9056) (#9085)
+  * Fix password checks on admin create/edit user (#9076) (#9081)
+  * Fix: add search to reserved usernames (backport #9063) (#9065)
```
Comment (techknowlogick) [inline CHANGELOG.md]:
```suggestion
   * Add search as a reserved username (#9063) (#9065)
```

**Review by techknowlogick — APPROVED** [review]

**Review by jolheiser — APPROVED** [review]

**Review by axifive — APPROVED** [review]

**Comment by jolheiser** [comment]
@6543 This needs to be ported to master, please. 😃

**Comment by 6543** [comment]
On the todo, cant do it now :(

**Comment by 6543** [comment]
@jolheiser feel free to cherry-pick it ;)

**Pull request merged**